.github/workflows/update.yaml

```diff
@@ -0,0 +1,42 @@
+name: build
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+  schedule:
+    - cron:  '0 0 * * 1'  # Every monday UTC 0:00 (JST 9:00)
```

Comment:
What is necessary or this is just a good opportunity to add that feature?

Reply:
This is for triggering weekly update of aports-ros-experimental based on the official rosdistro.